### DISCUSSION TITLE
Force HTTPS with TLS Certificate

### DISCUSSION THREAD

**Discussion by derrge** [ZoeyVid/NPMplus#538] [Q&A] [answered]
Hello I'm trying to proxy to my Adguard server which is a stand alone VM. I can proxy to it with out toggling on force HTTPS with a TLS Certificate. If anyone could point me in the right direction. I would appreciate it.

**Comment by Zoey2936** [comment]
what do you mean?

**Comment by derrge** [comment]
I attached some screen shots. if i select my certificate and toggle on force https it does not proxy to the host.
![adguard example](https://github.com/ZoeyVid/NPMplus/assets/70377649/b21e57ed-ba5f-4203-9043-2a5ab5e1fa4c)
![adguard example2](https://github.com/ZoeyVid/NPMplus/assets/70377649/b6bcf8d8-5397-48a2-867e-e1ead1052922)

**Reply by Zoey2936** [reply]
sorry, but I still don't understand what you mean / what your question/problem is

**Reply by derrge** [reply]
I'm sorry. I know I'm pretty bad at explaining things. If I browse to http://adguard.example it works without the letsencrypt certificates. If I browse to https://adguard.example it does not work after selecting my TLS Certificate and Forcing HTTPS.

**Reply by Zoey2936** [reply]
![grafik](https://github.com/ZoeyVid/NPMplus/assets/75573284/ab177002-b588-4f9a-b2eb-a7f1d66e92f0)
![grafik](https://github.com/ZoeyVid/NPMplus/assets/75573284/044c99fb-e5c9-417b-aa80-0bfd66c63fe8)
this are my settings

**Reply by derrge** [reply]
Sorry I probably have a misunderstanding on how nginx proxy manager works. Does it support proxying to services that are not running on the docker host?

**Reply by Zoey2936** [reply]
you can proxy to any IP, it doesn't need to be the docker host / localhost, but it shouldn't be the virtual ip of a docker container

**Reply by derrge** [reply]
Okay that's great. Then I'm at a total loss at what my issue is. I shouldnt need to configure any of the Encryption settings on my Adguard VM because that should all be taken care of by the Nginx Proxy Manager when I select my certificate and force SSL

**Reply by derrge** [reply, marked as answer]
Zoey thanks for the help and the patience. I figured out my issue. I had setup my local DNS rewrite record to the IP of the Adguard server and not the IP of the Nginx-Proxy-Manger. Once I corrected that misconfiguration everything works as it should.
